packages/quantic/force-app/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.js

```diff
@@ -0,0 +1,24 @@
+import { LightningElement, api } from 'lwc';
+
+export default class DescriptionStrengthIndicator extends LightningElement {
+    @api helpLabel = "Don’t know what to write?";
+    progression = 0;
+
+    get variant() {
+        return this.progression < 100 ? "warning" : "base-autocomplete";
+    }
+
+    get message() {
+        return this.progression === 0 ? "Provide details" : this.progression < 100 ? "Provide more details " : "Thank you!";
+    }
+
+    @api isFull() {
+        return this.progression >= 100;
+    }
+
+    @api progress() {
+        if (this.progression < 100) {
+            this.progression += 25;
+        }
+    }
```

Comment:
I think I would make it like this:

```suggestion
@track
_progression = 0;

@api
set progression(progression) {
  this._progression = progression;
}
get progression() {
  return this._progression;
}

@api
progress() {
  this._progression = (this._progression < 100) ? this._progression + 25 : this._progression;
}
```

Reply:
@erocheleau  Yess siir 👌 just one question here, Why did u use the @track decorator for the  progression field, isn't  just used to track changes in objects and arrays?  like documented here : https://lwc.dev/guide/javascript_reactive#track-changes-inside-objects-and-arrays